### ISSUE TITLE
Omniro is severely underpowered in his current state

### ISSUE THREAD

**Issue by DeckardCainRPC** [ChalkyBrush/roshpit-bug-tracker#1172]
**TL;DR VERSION SINCE YOU ARE ALLERGIC TO READING:**
Omniro lacks bonuses to Base Ability Damage, Item Damage, Elemental Damage, and any boosts to specific damage types; additionally, his native post-mit amp is not good, but that is not as severe a problem. His base damage values would be suitable for endgame content **IF, FUCKING IF,** he just had these specific percentile multipliers... along with some tweaks to his attack damage, which is remarkably low.

**AND HERE IS WHAT I SUGGEST YOU SHOULD DO TO FIX IT:**
1. Give this additional property to his E spell (yes, his E spell, because its tooltip is the shortest and therefore has the most room for more text): "Additionally, all gear rolls that enhance Elemental damage are multiplied. This bonus is recalculated when changing gear and upon respawning." This property scales up with E rank,

"GEAR BONUS MULTIPLIER" "1/2/3/4/6/8/10"

The +1000% Elemental Damage rolls on stuff like Val'dun's hat or the Serengaard rock will now be even more significant considerations, especially since they basically fucking sacrifice a shit ton of rune rolls in that item slot, and Omniro thrives on runes.

All other heroes with T4 runes that give stat-based Elemental% frequently give 20k%+ to the element(s) in question, so this is still a less-than-status-quo boost. Normal has the unique distinction of being the only element with no "neutral" (all-hero) gear that increases its damage - Normal% only appears on a few Immortal weapons, and bonuses to "All Elemental Damage" exclude Normal.

2. Give this additional property to his R spell: "Additionally, Omniro passively deals increased damage with equipped Items." This property scales up with R rank, +100/200/300/400/500/600/700%.

At rank 7 that's just a free Steamboots with ~24k AGI, or like one Tanari Waterstone for each of his two Faceless Testicles. It's not even close to OP, especially considering his relatively low Elemental% boosts if you implement the E change above.

3. Even with EXTREME min-maxing, his sorry ass can only get up to around 80m total attack power, and that's with him wearing Dark Reef Shark Helmet while standing near a Sephyr with the highest number of Wolfir Druid Spirit Helm creep auras he can possibly stack. Compare this to the ~400m "decent" baseline everybody else operates with. My suggested solution for this is to double W4's Mace effect like a lazy asshole, and then change Fire's Mace effect to this:

"Adds attack damage based on a multiple of Omniro's total rune points. The stated value is multiplied by the level of [Q1]."

The base value is 10. Have ITEM_UNIT or RUNE_UNIT or whatever just check his total rune points on gear equip and on respawn, then multiply that number by 10 and then by [Q1] if [Q1] > 0, and that's how much base attack damage the Fire Mace buff would give. With 20 points in Q1 and no other rune points, this would only give 200 * 20 = 4k base attack damage. With 20 Q1 and all 241 rune points assigned while standing naked, this would give 48,200 base attack damage, a significant improvement from the 20k it would give in the current patch. A *chungus giganticus* Omniro with 900 Q1 would still only get 8.1m base attack damage, which is... a neat reward for Wario-esque supervillains (Xhul) who actually manage to set that up, but still not even close to OP. The point is, this change lets Omniro players invest in whatever runes they want, while significantly reducing the hero's soul-crushingly forced reliance on committing a bunch of slots to Q1 because Chalky is a goon. Investing in the rune gently is still a great value multiplier if you're focusing a different rune (Shadow, Demon, Poison) whose damage is very reliant on total attack power.

**THERE WAS NO TL;DR VERSION LOL I TRICKED YOU**

**Comment by ChalkyBrush** [comment]
> THERE WAS NO TL;DR VERSION LOL I TRICKED YOU

What the fuck!!

**Comment by DeckardCainRPC** [comment]
[11:18 AM] Deckard Cain: redledcat you mentioned that these changes might make his immo glyph obsolete
[11:18 AM] Deckard Cain: i agree, if it's handled as bonus elemental%, then it would be pointless
[11:18 AM] Deckard Cain: so i would recommend changing it to a miscellaneous multiplier, non-BAD and non-Element
[11:19 AM] Deckard Cain: just another, similar pre-mit amp mod
[11:19 AM] Deckard Cain: 15x pre-mit amp to your "main cannon" is more than acceptable, for people running 4-element rotations

**Comment by redledcat** [comment]
What is truly catastrofical, it's recharge time.

**Comment by redledcat** [comment]
![image](https://user-images.githubusercontent.com/32621322/57397921-73b9b080-71d6-11e9-8921-c98ad229906e.png)
best I got with E2
1)recharges due to dragon element
2)Q1 gives too low base damage / E2 + phoenix not fine too, so I got farseers +divinex
30
3)About 2-3 minutes for paladin boss, about 10% damage to sven

**Comment by DeckardCainRPC** [comment]
blinded by turkish science
